src/common/transformations/src/transformations/common_optimizations/compress_float_constants.cpp

```diff
@@ -25,23 +27,30 @@ std::shared_ptr<ov::Node> change_constant_precision_to_fp16(std::shared_ptr<ov::
     if (dst_data == nullptr)
         return nullptr;
 
-    bool is_overflow = false;
+    int num_out_of_range = 0;
     for (size_t i = 0; i < size; ++i) {
```

Comment:
can we parallelize this `for`? we can accelerate loading more.
@nshchego, @dmitry-gorokhov, please advice.

Reply:
@pavel-esir. we can consider this optimization in separate PR but it really makes sense